### DISCUSSION TITLE
how do i connect another app on another server to the same mysql database please?

### DISCUSSION THREAD

**Discussion by ivan006** [nocodb/nocodb#674] [Discord Threads]

**Comment by o1lab** [comment]
You can connect multiple databases from same nocodb docker already.

Moving this to discussion

**Reply by ovizii** [reply]
As far as I understand you can have App1 with its DB1 and then simply connect nocodb to DB1 without interfering with App1. Pretty smooth, you can just access some other app's DB if you know what you are doing and use nocodb functionality on top of it.

**Reply by ovizii** [reply]
> You can connect multiple databases from same nocodb docker already.

But not at the same time, right? 
I was looking for something like airtable seems to  offer, perhaps in the future could do the same too?

![image](https://user-images.githubusercontent.com/837902/148965102-2c278f63-bd88-4b6e-bb87-c1c9fbfb7bc6.png)

**Reply by o1lab** [reply]
Ah this is more like pulling data from other source and putting it to airtable. This is something that can be done.

**Reply by o1lab** [reply]
That can be done in future :)
